packages/select/src/FilterMultiSelect/components/Trigger/TriggerButtonBase/TriggerButtonBase.tsx

```diff
@@ -10,11 +10,13 @@ import styles from "./TriggerButtonBase.module.scss"
 export type TriggerButtonBaseProps = {
   children: React.ReactNode
   classNameOverride?: string // TODO: migrate it to use OverrideClassName<T> and omit the props controlled by React-Aria
+  disabled?: boolean
 }
```

Comment:
Hey @kaydwithers now this is being used independently we decided its best to extend native button props instead of adding only disabled, ie:
```
export type TriggerButtonBaseProps = {
  children: React.ReactNode
  classNameOverride?: string
} & ButtonHTMLAttributes<HTMLButtonElement>
```

We can then spread it into the TriggerButtonBase below the `buttonProps` spread. It'll need to be below here to make sure anything passed in overrides what React-Aria provides.

Reply:
Our convention for spreading native props is to use `...restProps` underneath what's been destructured.

Reply:
Thanks @mcwinter07, that's done now.